zhaquirks/sonoff/s60zbtpf.py

```diff
@@ -0,0 +1,154 @@
+"""Sonoff S60ZBTPF - Zigbee Smart Plug."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfPower,
+)
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl import ClusterType, foundation
+import zigpy.zcl.foundation as zcl_f
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class SonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+    manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        outlet_control_protect_setting = ZCLAttributeDef(
+            name="outlet_control_protect_setting",
+            id=0x7007,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload_enable = ZCLAttributeDef(
+            name="ac_current_max_overload_enable",
+            id=0x700C,
+            type=t.uint8_t,
+        )
+        ac_voltage_max_overload_enable = ZCLAttributeDef(
+            name="ac_voltage_max_overload_enable",
+            id=0x700E,
+            type=t.uint8_t,
+        )
+        ac_power_max_overload_enable = ZCLAttributeDef(
+            name="ac_power_max_overload_enable",
+            id=0x7010,
+            type=t.uint8_t,
+        )
+        ac_current_max_overload = ZCLAttributeDef(
+            name="ac_current_max_overload",
+            id=0x700D,
+            type=t.uint32_t,
+        )
+        ac_voltage_max_overload = ZCLAttributeDef(
+            name="ac_voltage_max_overload",
+            id=0x700F,
+            type=t.uint32_t,
+        )
+        ac_power_max_overload = ZCLAttributeDef(
+            name="ac_power_max_overload",
+            id=0x7011,
+            type=t.uint32_t,
+        )
+
+    class ServerCommandDefs(zcl_f.BaseCommandDefs):
+        """Server command definitions."""
+
+        self_test: Final = zcl_f.ZCLCommandDef(
+            id=0x00, schema={"identify_time": t.uint8_t}, direction=False
+        )
+
+    @property
+    def _is_manuf_specific(self):
+        return False
```

Comment:
This is not needed now that `manufacturer_id_override: t.uint16_t =  ..` is used.